web/wp-content/themes/dctx-v2/functions.php

```diff
@@ -0,0 +1,241 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-htq7yq.php, but found phpcs-scan-Htq7yq.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).